src/examples/CMakeLists.txt

```diff
@@ -33,7 +33,14 @@ add_executable(	dlt-example-filetransfer ${dlt_example_filetransfer_SRCS})
 target_link_libraries(dlt-example-filetransfer dlt )
 set_target_properties(dlt-example-filetransfer PROPERTIES LINKER_LANGUAGE C)
 
-install(TARGETS dlt-example-user dlt-example-user-func dlt-example-user-common-api dlt-example-filetransfer
+set(dlt-example-multicast-clientmsg-view_SRCS dlt-example-multicast-clientmsg-view.c)
```

Comment:
This only works if WITH_UDP_CONNECTION is set to ON, right? Then these lines should be put under if (WITH_UDP_CONNECTION) condition.

Reply:
Done